### ISSUE TITLE
RFID Tags?

### ISSUE THREAD

**Issue by kriswiner** [markreds81/RFID#2]
Which RFID tags did you use to test this? I tried [these](https://www.amazon.com/YARONGTECH-Proximity-125KHZ-Control-Plastic/dp/B01M1L7NHI/ref=pd_bxgy_sccl_1/131-7845572-7024060?pd_rd_w=cSobF&content-id=amzn1.sym.6ab4eb52-6252-4ca2-a1b9-ad120350253c&pf_rd_p=6ab4eb52-6252-4ca2-a1b9-ad120350253c&pf_rd_r=TKYP2MWPY9KKRC0NQVRM&pd_rd_wg=HkS7S&pd_rd_r=bf20e0ab-e612-4e2e-841f-cf13f0029925&pd_rd_i=B01M1L7NHI&th=1) but I can't get any output on the serial monitor...If you can share a link to tags that work with your sketch I would ppreciate it greatly, thanks!

**Comment by markreds81** [comment]
These [ones](https://www.cemitonline.com/en/products/electronic-keys/) works with my application. They are 125 Khz desfire 40 bits.

**Comment by kriswiner** [comment]
Great, thanks! Any idea where I could buy some to test?

**Comment by markreds81** [comment]
Yes, sure. Write to info@cemitonline.com

**Comment by kriswiner** [comment]
Thanks. One last question if I might:

Is the MLX90109 finicky in this regard, meaning are there only certain
types of 125 kHZ RFID tags it can read?

I don't know much about LF RFID (obviously) but I am surprised there are
125 kHz RFID tags the MLX90109 evl kit cannot read....do you know of a link
where this is discussed?

**Comment by markreds81** [comment]
As far as I know, I don't see such a thing but I'm not an expert. Have you already took a read on the module documentation?

**Comment by kriswiner** [comment]
Yes, I have read all of the MLX90109 and EVK90109 documents many times.
Running your basic code and connecting the EVK90109 properly to a MCU is
not that hard to do. It should work but it doesn't. I am trying to figure
out why not. If only a small number of 125 kHz RFID tags will work with the
MLX90109, then this would explain my results.

I only have one type of RFID tag at the moment. I want to avoid buying
several types just to find only certain ones will work or none of them will
work. I am surprised that any of them do not work....

Anyway, thanks for your help.

**Comment by markreds81** [comment]
Hi @kriswiner, we have encoutered the same problem in these days because we changed our tag supplier. After further investigations  we think it depends from tag's antenna geometry. In our tests with different tags it change the minimum distance needed by the reader to decode the tag's ID.
We are continuing our internal tests.
